documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,575 @@
+proposal from the ground up. In that case, we will follow this process again
+from the start with the new proposal, so we will end the process for the first
+proposal at this round and wait for a new one to be raised.
+
+Regardless of the outcome, the author is responsible for completing
+[the steps to finalise the discussion](#outcomes).
+
+### Objection revision round
+
+This round, similar to the previous revision round, lasts as long as the author
+needs to completely address the paramount objections. The author will work with
+the person who raised the objection to revise the proposal to address the issue
+to resolution. The goal of this round is to reach a "good enough" middle ground
+solution that either completely addresses the paramount objection or mitigates
+the potential harm identified.
+
+As stated in the previous round's description, objection revision should only
+happen if paramount objections can indeed be revised to remove the harm. If the
+paramount objection involves a core issue of the proposal that cannot be
+revised, then an entirely new proposal should be written, rather than the
+"revision" replacing the entire existing proposal. It may be the case that in
+the process of revision, the author of the proposal realises that they need to
+"return to the drawing board". Please keep in mind that this is not a failure
+and should be celebrated. So long as the objections were truly paramount, the
+participants in the discussion and the author of the proposal have successfully
+prevent long-term harm to the goals or people on the project. That is good news.
+
+Once the paramount objections are addressed, we will consider the proposal to be
+accepted. The team as a whole can move on the next step of the project, whether
+that is implementation planning or implementation itself.
+
+## Outcomes
+
+There are two possible outcomes for any proposal that follows this process:
+approval or tabling. In both cases, the author is expected to:
+
+- Update the current round callout to indicate the outcome
+- If the proposal is in a GitHub discussion, lock the discussion
+- If the proposal is in a PR, merge the PR if approved or close it if tabled
+- Move the associated card on the discussion dashboard to the appropriate column
+
+Further requirements specific to each outcome follow.
+
+### Approval
+
+Approval occurs once all paramount objections have been addressed and the author
+is satisfied with the state of the proposal. At this point any necessary GitHub
+issues should be created and linked to the proposal.
+
+### Tabling
+
+Tabling can occur in the following cases:
+
+- At any time, at the discretion of the author
+- Before a proposal is written, if the team reneges on a requested discussion
+- After a proposal is written, but before the discussion happens, if it is clear
+  the proposal will not fit the needs of the project
+- During the objection round when insurmountable paramount objections are raised
```

Comment:
I'm not sure I understand the question.

If you're asking whether the fact that the author can table the discussion at their discretion at any time, I think it so. If someone no longer believes in their proposal I don't think it makes sense to ask them to continue to clarify, revise, and respond to objections with respect to it. If another person wishes to take it up they can base their proposal on the existing text and start a new decision-making process for it.

If you're asking whether all four points can be summarised as "at the author's discretion", then I think it's _technically_ true. But I also think clarifying specific, expected situations where tabling would occur is warranted. Not all of these points are necessarily initiated by the author. It could be that an external stakeholder decides a project is no longer necessary, causing the third case to happen. I wouldn't say that's the "author's discretion" but is a realistic and plausible case that could arise.